tests/components/unifiprotect/test_repairs.py

```diff
@@ -357,3 +357,64 @@ async def test_rtsp_writable_fix(
     ufp.api.update_device.assert_called_with(
         ModelType.CAMERA, doorbell.id, {"channels": channels}
     )
+
+
+async def test_rtsp_writable_fix_when_not_setup(
+    hass: HomeAssistant,
+    ufp: MockUFPFixture,
+    doorbell: Camera,
+    hass_client: ClientSessionGenerator,
+    hass_ws_client: WebSocketGenerator,
+) -> None:
+    """Test RTSP disabled warning if the integration is no longer setup."""
+
+    for channel in doorbell.channels:
+        channel.is_rtsp_enabled = False
+
+    await init_entry(hass, ufp, [doorbell])
+    await async_process_repairs_platforms(hass)
+    ws_client = await hass_ws_client(hass)
+    client = await hass_client()
+
+    new_doorbell = deepcopy(doorbell)
+    new_doorbell.channels[0].is_rtsp_enabled = True
+    ufp.api.get_camera = AsyncMock(side_effect=[doorbell, new_doorbell])
+    ufp.api.update_device = AsyncMock()
+    issue_id = f"rtsp_disabled_{doorbell.id}"
+
+    await ws_client.send_json({"id": 1, "type": "repairs/list_issues"})
+    msg = await ws_client.receive_json()
+
+    assert msg["success"]
+    assert len(msg["result"]["issues"]) > 0
+    issue = None
+    for i in msg["result"]["issues"]:
+        if i["issue_id"] == issue_id:
+            issue = i
+    assert issue is not None
+
+    # Unload the integration to ensure the fix flow still works
+    # if the integration is no longer setup
+    await hass.config_entries.async_unload(ufp.entry.entry_id)
+    await hass.async_block_till_done()
```

Comment:
It's probably *fine* to leave it like this, but other than these two lines, the two tests are identical.

Reply:
I think the current practice is to avoid any `if`/branching in the test body so I copied the test

Reply:
Yeah, wasn't sure if it's allowed for something like this. This is good to go IMO.